src/Microsoft.AspNet.Mvc.Razor/IRazorPage.cs

```diff
@@ -3,10 +3,10 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Threading.Tasks;
 using Microsoft.AspNet.Mvc.Rendering;
-using Microsoft.AspNet.PageExecutionInstrumentation;
```

Comment:
Do we need the `Microsoft.AspNet.PageExecutionInstrumentation.Interfaces` project anymore? Looks like you've removed all `IBufferedTextWriter`, `IPageExecutionContext`, and `IPageExecutionListenerFeature` references.